CHANGELOG.md

```diff
@@ -7,18 +7,18 @@
 #### New roles
 
   - addons/Lmod: allow to install Lmod and specify custom modulefiles path (#390)
-  - addons/singularity: allow to instal Singularity (#403)
+  - addons/singularity: allow to install Singularity (#403)
   
 #### Roles improvement
 
   - core/ssh_master:
     - add ssh jump capability in multi icebergs context (one level only) (#395)
   - addons/nic_nmcli:
-    - adapt to new inventory format (#401)
+    - convert to new inventory format (#401)
   - core/hosts_file:
-    - prevent many blanck files when hosts without network_interfaces (#406)
-  - core/rsyslog_server/client:
-    - allows custom server port (#397)
+    - prevent many blank lines when hosts without network_interfaces (#406)
```

Comment:
I missed it yesterday: _when hosts have no network_interfaces in the inventory_ ?